packages/edge/README.md

```diff
@@ -0,0 +1,78 @@
+
+# React Native AEP Edge Network
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepedge.svg)](https://www.npmjs.com/package/@adobe/react-native-aepedge) 
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepedge)](https://www.npmjs.com/package/@adobe/react-native-aepedge)
+
+## Usage
+
+### Initializing:
+
+Initializing the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](https://aep-sdks.gitbook.io/docs/getting-started/get-the-sdk#2-add-initialization-code). 
+
+Example:
+
+Objective-C
+```objectivec
+[AEPMobileCore registerExtensions: @[AEPMobileLifecycle.class, AEPMobileEdgeIdentity.class, AEPMobileEdge.class] completion:^{
+          [AEPMobileCore lifecycleStart:@{@"contextDataKey": @"contextDataVal"}];
+```
+
+JAVA
+```java
+  Edge.registerExtension();
+```
```

Comment:
Should I include Identity.registerExtension();  here since they should go together?

Reply:
No, Edge Identity is sufficient here.